testsuite/openshift/objects/authorino.py

```diff
@@ -50,6 +51,8 @@ def create_instance(cls, openshift: OpenShiftClient, name, image=None,
                 "enabled": True,
                 "certSecretRef": {"name": listener_certificate_secret}
             }
+        if specific_name:
+            model["metadata"]["name"] = specific_name
```

Comment:
You can use `name` attribute instead of creating a new one that is for same thing.

Reply:
For 99% of testcases is OK to have Authorino with blamed name as it is generated here https://github.com/Kuadrant/testsuite/blob/main/testsuite/tests/kuadrant/authorino/conftest.py#L34
Only this testcase needs to have it. But you are right that this is not good solution, so I've found better place to set the Authorino name here testsuite/tests/kuadrant/authorino/conftest.py

Reply:
You shouldnt change it in upper conftest either change it properl here and it cannot be done then we need to create a way how to do it here